src/components/TeamMemberTasks/TeamMemberTasks.jsx

```diff
@@ -122,68 +144,76 @@ const TeamMemberTasks = (props) => {
               </Table>
             </td>
             <td>
-              <Table borderless className='team-member-tasks-subtable'>
+              <Table borderless className="team-member-tasks-subtable">
                 <tbody>
                   {user.tasks &&
-                    user.tasks.map((task, index) => (
-                      task.wbsId && task.projectId &&
-                      <tr key={`${task._id}${index}`} className='task-break'>
-                        <td className='task-align'>
-                          <p>
-                            <Link to={task.projectId ? `/wbs/tasks/${task._id}` : '/'}>
-                              <span>{`${task.num} ${task.taskName}`} </span>
-                            </Link>
-                            {
-                              task.taskNotifications.length > 0 &&
-                              <FontAwesomeIcon
-                                className="team-member-tasks-bell"
-                                icon={faBell}
-                                onClick={() => {
-                                  handleOpenTaskNotificationModal(user.personId, task, task.taskNotifications);
-                                }}
-                                />
-                              }
-                          </p>
-                        </td>
-                        {task.hoursLogged != null && task.estimatedHours != null &&
-                          <td className='team-task-progress'>
-                            <div>
-                              <span>
-                                {`${parseFloat(task.hoursLogged.toFixed(2))}
```

Comment:
Hi Kaung, I noticed that this orange part seems to be exactly the same as the blue part below except that the blue part moves `(task, index) =>` to a new line, is this done by the code formatter?

Reply:
Hi Bruce, yes I believe this is done by the code formatter.